demos/src/Provisioning/aws_iot_demo_provisioning_with_keys_and_cert.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ * Copyright (C) 2019 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
What's the rationale for this change?

Reply:
The rationale is that the library was developed last year, and the year in the copyright was a copy-pasta error